allensdk/brain_observatory/behavior/data_objects/licks.py

```diff
@@ -122,3 +123,77 @@ def to_nwb(self, nwbfile: NWBFile) -> NWBFile:
         nwbfile.add_processing_module(licks_mod)
 
         return nwbfile
+
+
+class VBNLicks(Licks):
+
+    @classmethod
+    def from_json(cls, dict_repr: dict) -> "VBNLicks":
```

Comment:
`from_json` is not needed. The `BehaviorStimulusFile` and `SyncFile` will just be passed to `from_stimulus_file` from the session class.

Reply:
So the expectation is that the session class will create and load the sync and behavior data and then pass them down to the data object classes?